### DISCUSSION TITLE
new function

### DISCUSSION THREAD

**Discussion by Ga3eta** [BartoszCichecki/LenovoLegionToolkit#936] [Ideas]
Hello, here are some ideas for the program:

1) Add a restart of the video driver of the video card after startup (program, system, a certain application, with a time delay after startup, etc. - give a choice) according to my observations, after this procedure, the Nvidia driver, especially on the Ampere chip, works more stable, because without Reloads vomit black screens and frame rate drops, the latency also falls, after observing in Latencymon. With what it is connected without a clue, the initialization of the driver at startup may work crookedly.. There is an example that comes with the CRU program as a separate file. But it does not implement automation. 
2) Try all the same to implement the function of controlling the curvature of the cooling system in laptops in 2021, because the cooling system stupidly does not have time to cool the laptop during turbo boost until the turbines are fully accelerated

**Comment by BartoszCichecki** [comment]
1. You can try "Restart GPU" in Actions, otherwise you can make your own batch file that will do what you want and run it with Actions. But it rather sounds like an issue that you have with drivers and this is just a workaround.
2. Not possible, discussed many times, also mentioned in readme.
